### ISSUE TITLE
Is it ok to NOT commit the whole packages directory of a Visual Studio solution nowadays?

### ISSUE THREAD

**Issue by drauch** [NuGet/Home#7038]
https://github.com/github/gitignore/pull/1936#issuecomment-398019528 is trying to determine whether the whole packages sub directory of a Visual Studio solution can be safely git-ignored or whether some content is important to checkin.

I know that in the earlier days it was important to checkin "repositories.config", however, it is no longer the case. Is there any other content (e.g., a "build" sub directory ?!) which has to be checked in?

TL;DR: Is it safe to ignore the whole packages sub directory of a Visual Studio solution?

Best regards,
D.R.

**Comment by rohit21agrawal** [comment]
any config files , like nuget.config or repositories.config should be checked in, otherwise everything else can be safely ignored.
